sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_api_versions.py

```diff
@@ -27,13 +27,13 @@ def validate_api_version(api_version: str, client_kind: str) -> None:
 
     if client_kind == "form":
         try:
-            api_version = FormRecognizerApiVersion(api_version)
```

Comment:
Doing this to not modify the original API version string that is appended to the error message. This will keep the string untouched in the error message.